README.md

```diff
@@ -1,29 +1,26 @@
-# Axis - The Event-Based Dependency Manager for Kubernetes
+# Argo Events - The Event-Based Dependency Manager for Kubernetes
 [![Build Status](https://travis-ci.org/blackrock/axis.svg?branch=master)](https://travis-ci.org/blackrock/axis) [![License Apache 2.0](https://img.shields.io/badge/License-Apache2-brightgreen.svg)](https://img.shields.io/badge/License-Apache2-brightgreen.svg)
```

Comment:
We probably want to change these badges as well. Also we want to remove the `.travis-ci.yml` and put in a `.argo-ci/ci.yaml` file instead.

Reply:
We should discuss with Jesse and Alexander.

Reply:
okay, we'll tackle this outside this PR